src/genworld_gui.cpp

```diff
@@ -878,6 +878,24 @@ void StartNewGameWithoutGUI(uint32 seed)
 	StartGeneratingLandscape(GLWM_GENERATE);
 }
 
+/**
+ * Start a normal game from a heightmap file without the GUI.
+ * @param seed The seed of the new game.
+ */
+void StartNewHeightMapGameWithoutGUI(uint32 seed)
+{
+	/* GenerateWorld takes care of the possible GENERATE_NEW_SEED value in 'seed' */
+	_settings_newgame.game_creation.generation_seed = seed;
+
+	uint x = 0;
+	uint y = 0;
+
+	/* If the function returns negative, it means there was a problem loading the heightmap */
```

Comment:
Comment says <0. Code says != 0.

Reply:
Comment inherited from original patch, actually.
I guess the author, by 'negative', meant the boolean meaning as in 'not true', not the integer one.

I'll rephrase that